src/frameworks/flutter/utils.ts

```diff
@@ -8,3 +10,27 @@ export function assertFlutterCliExists() {
       "Flutter CLI not found, follow the instructions here https://docs.flutter.dev/get-started/install before trying again.",
     );
 }
+
+export async function getTreeShakeFlag(): Promise<string> {
+  const pubSpecPath = "./pubspec.yaml";
+  let pubSpec: Record<string, any> = {};
+  try {
+    const pubSpecBuffer = await readFile(pubSpecPath);
```

Comment:
we should use a path relative to the project root as the project can be located in a subdir, as we do in discover https://github.com/firebase/firebase-tools/blob/cc6744f3e2bbd4fae0d08b1bf4ef999ce5c22fe0/src/frameworks/flutter/index.ts#L16-L20